src/applications/disability-benefits/all-claims/submit-transformer.js

```diff
@@ -474,7 +474,10 @@ export function transform(formConfig, form) {
     addForm0781,
     addForm8940,
     addFileAttachmments,
-  ].reduce((formData, transformer) => transformer(formData), form.data);
+  ].reduce(
+    (formData, transformer) => transformer(formData),
+    _.cloneDeep(form.data),
```

Comment:
Unrelated to this PR, but there was a bug in the submit step caused by some data mutation. We should be operating on a clone of the form data.

Reply:
Additional context: Mark found that when he submitted the form, it would fail as expected (back end issues; the form data submitted was fine), but one of the review page's chapter accordions grew an alert mysteriously. We were on a call for a while to try and figure out what was causing it, and eventually settled on some data mutation in the submit transformer.

It's not clear _what's_ being mutated, but we figure as long as we start the transformers with a clone, we're okay. It's not ideal that at least one of those transformers are mutative, but it's not really the end of the world either. :man_shrugging: